.gitignore

```diff
@@ -91,3 +91,5 @@ user_song_list.h
 compile_commands.json
 .clangd/
 .cache/
+
+.vscode/
```

Comment:
hi, please clean this up. keyboard PR's should only affect the keyboard folder. thanks

Reply:
Removed those lines in the latest commit